### ISSUE TITLE
Water meter readings

### ISSUE THREAD

**Issue by greenwell72** [AdyRock/com.linktap#20]
Hi, thank you for this great app. I have a question about the water meter reading which it extracts. From what I can see it returns the last flow amount as opposed to working as a counter. For example if I have two scheduled watering events per day where for example the first is 25l and the second 35l. At the end of the day i will see a water meter value of 35l rather than 60l (taking just one day  usage as an example). I would have assumed this would be a cumulative number since the device was used.

**Comment by AdyRock** [comment]
I agree that would be nice but the app just reports the data that is sent from LinkTap.
I will have a look to see if I can add some extra parameters that are generated by the app to sum the session data over a day / total.

**Comment by greenwell72** [comment]
Thanks Adrian that would be great. I can also contact LinkTap to see if this is something the intend changing.

**Comment by AdyRock** [comment]
The new test version now has the accumulated water volume. It is maintained by the app so it will start at zero when you updated to this version.

**Comment by greenwell72** [comment]
Brilliant thank you. Will test shortly.
